internal/cmd/commands/database/funcs.go

```diff
@@ -25,6 +25,8 @@ func migrateDatabase(ctx context.Context, ui cli.Ui, dialect, u string, initiali
 		ui.Error(fmt.Errorf("Error establishing db connection: %w", err).Error())
```

Comment:
I kept this `SqlOpen` in this function because the whole connection is scoped to this function, and it means we can safely defer the close. Moving it out of this scope makes the closing a little messier (and we have to do it everywhere this function is called).